warehouse/templates/pages/help.html

```diff
@@ -152,6 +154,17 @@ <h2>{{ project_name_claim() }}</h2>
         </p>
       </section>
 
+      <section id="file_size_limit" class="common-question">
+        <h2>{{ file_size_limit() }}</h2>
+        <p>
+          If you can't upload your release to PyPI because you're hitting the upload file size limit, please <a href="https://github.com/pypa/warehouse/issues/new">file an issue</a> and tell us:</p>
+        <ul>
+          <li>The name of the project</li>
+          <li>The size of your release (example: 300 MB)</li>
```

Comment:
300MB is kind of on the larger end of our normal requests. Instead of providing an example we could just say:

> The size of your release, in megabytes

Reply:
Got it. Fixing.